source/configure/configuration-settings.rst

```diff
@@ -5229,6 +5229,18 @@ For MySQL Group Replication, the absolute lag can be measured from the number of
 
    select member_id, count_transactions_remote_in_applier_queue FROM performance_schema.replication_group_member_stats where member_id=<>
 
+Image Settings
+~~~~~~~~~~~~~~
+
+Maximum Image Resolution
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Maxiumum image resolution size for message attachments in mega pixels. This setting can only be changed from ``config.json`` file, it cannot be changed from the System Console user interface.
+
++---------------------------------------------------------------------------------------------------------------+
+| This feature's ``config.json`` setting is ``"FileSettings.MaxImageResolution": 33`` with numerical input.     |
```

Comment:
@streamer45 - Can you please confirm that the default image resolution configured is correct as documented (and that the ``config.json`` file accepts the value for this config setting in mega pixels, rather than pixels). Thank you!

Reply:
The default is 8K resolution (7680x4320 pixels) which is roughly 33MPx. The config value is in pixels so the default is `33177600`.